sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/promotion/passwords/SyncPasswordsPromotion.kt

```diff
@@ -51,6 +52,7 @@ import kotlinx.coroutines.flow.launchIn
 import kotlinx.coroutines.flow.onEach
 
 @ContributesMultibinding(scope = AppScope::class)
+@PriorityKey(200)
```

Comment:
Why did we go with 200? Is there a `PasswordsScreenPromotionPlugin` with priority 100? (Should it be a constant?)

Reply:
Good idea, and i see now that's the pattern for other priority key definitions. Done!